src/TextField/TextField.jsx

```diff
@@ -53,42 +53,48 @@ const styles = theme => {
  * />
  * ```
  */
-export const TextField = ({
-  InputProps,
-  autoComplete,
-  disabled,
-  error,
-  fullWidth,
-  helperText,
-  id,
-  label,
-  required,
-  ...other
-}) => {
-  const helperTextId = helperText && id ? `${id}-helper-text` : undefined
-
-  // Sugar for setting autoComplete using a boolean value.
-  if (isBoolean(autoComplete)) {
-    autoComplete = autoComplete ? 'on' : 'off'
-  }
+class TextField extends React.Component {
+  render () {
+    const {
+      disabled,
+      error,
+      fullWidth,
+      helperText,
+      id,
+      InputProps,
```

Comment:
Some disagreement over how to sort capital letter IDs here. I think the general convention observed in the project is to sort capital letters first 😬

Reply:
From their docs:
![Screen Shot 2019-04-09 at 3 46 08 pm](https://user-images.githubusercontent.com/127084/55775998-9cb72a80-5ade-11e9-9082-c487ff00107c.png)

Reply:
WHUT

Reply:
TIL that capitalised props go first before non-capitalised props.